### PR TITLE
Submitting materials for lesson09

### PR DESCRIPTION
**Pull request by ptfbanks**
Submitting materials for lesson 09.

### CODE REVIEW

File: students/ptfbanks/Lesson09/obj_mailroom.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    def add_donor(self, donor):
+        self.donors.append(donor)
+
+    def get_all_donor_names(self):
+        return [donor.full_name for donor in self.donors]
+    
+    def thank_you(self):
+        donor_name = None
+        while not donor_name:
+            donor_name = name_input()
```
Comment (natasha-aleksandrova) [inline students/ptfbanks/Lesson09/obj_mailroom.py]: you shouldn't have any `input` calls in your data classes (even though it's not used directly)

File: students/ptfbanks/Lesson09/obj_mailroom.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                donor_file.write("Thank you {}, for your generous donation of {}!".format(donor.full_name, donor.donations))
+
+d1 = Donor('Bob', 'Williams', [5580.00, 3245.50, 1000])
+d2 = Donor('Tom', 'Haskell', [345.05, 3245.90, 243.05])
+d3 = Donor('Earl', 'Jackson', [4553.45, 45.6, 345.00])
+d4 = Donor('Les', 'Thomas', [34.00, 325.00, 3665.00])
+d5 = Donor('James', 'Black', [154, 1300, 21])
+d6 = Donor('Ed', ' Jones', [35, 1])
+
+dh = DonorHistory([d1, d2, d3, d4,d5, d6])
```
Comment (natasha-aleksandrova) [inline students/ptfbanks/Lesson09/obj_mailroom.py]: 👍

**Comment by ptfbanks** [comment]
Thank you for your patience in my last dash to complete assignments 7, 8.
and 9.

I do not see these having been given credit.  Do I need to finalize
anything?

Thanks.

**Comment by natasha-aleksandrova** [comment]
Lesson 7 was graded, Lesson 8 was in the queue and I have now submitted.. Lesson 9 submission is missing. Can you please take care of that ASAP?

**Pull request merged**